projects/tinkerbell/tinkerbell-chart/helm/sedfile.template

```diff
@@ -0,0 +1 @@
+s,0.1.3,${IMAGE_TAG},g
```

Comment:
what does this get used for? 
Can we generate this dynamically so we don't have to keep another copy of `GIT_TAG`?

Reply:
The build version is set to ${GIT_TAG}-${BUILD_NUMBER}

Reply:
```
tlhowe@3c22fb41240e tinkerbell-chart % diff -qr old new
Files old/tinkerbell-chart/Chart.yaml and new/tinkerbell-chart/Chart.yaml differ
Files old/tinkerbell-chart/requires.yaml and new/tinkerbell-chart/requires.yaml differ
tlhowe@3c22fb41240e tinkerbell-chart % diff old/tinkerbell-chart/Chart.yaml new/tinkerbell-chart/Chart.yaml 
6c6
< version: 0.1.3
---
> version: 0.1.3-c7c82c3bfc0dd37a3f30adc7cc09feafb2c72980
tlhowe@3c22fb41240e tinkerbell-chart % diff old/tinkerbell-chart/requires.yaml new/tinkerbell-chart/requires.yaml 
4c4
<   name: tinkerbell/tinkerbell-chart-0.1.3
---
>   name: tinkerbell/tinkerbell-chart-0.1.3-c7c82c3bfc0dd37a3f30adc7cc09feafb2c72980
tlhowe@3c22fb41240e tinkerbell-chart %
```

Reply:
for dev builds, you get the github sha, but for prod builds, the code build build number

Reply:
awesome, let's merge this then!